docs/gettingstarted/docker-compose-MacOS.md

```diff
@@ -1,72 +1,56 @@
 ```
 
-* NOTE:  Be sure to account for disk space requirements for the docker volume created above. This volume is located in
-`/var/lib/docker/volumes/` and could grow significantly if there is an extended outage to the SC4S destinations
-(typically HEC endpoints). See the "SC4S Disk Buffer Configuration" section on the Configuration page for more info.
-
-* IMPORTANT:  When creating the directories below, ensure the directories created match the volume mounts specified in the
-`docker-compose.yml` file (if used).  Failure to do this will cause SC4S to abort at startup.
-
-* Create subdirectories `/opt/sc4s/local` `/opt/sc4s/archive` `/opt/sc4s/tls` 
+4. Create the subdirectories: `/opt/sc4s/local`, `/opt/sc4s/archive`, and `/opt/sc4s/tls`. If you are using the `docker-compose.yml` file, make sure these directories match the volume mounts specified in`docker-compose.yml`.
 
-* Create a file named `/opt/sc4s/env_file` and add the following environment variables and values:
+5. Create a file named `/opt/sc4s/env_file`.
 
 ```dotenv
 --8<--- "docs/resources/env_file"
 ```
-
-* Update `SC4S_DEST_SPLUNK_HEC_DEFAULT_URL` and `SC4S_DEST_SPLUNK_HEC_DEFAULT_TOKEN` to reflect the correct values for your environment.  Do _not_ configure HEC
-Acknowledgement when deploying the HEC token on the Splunk side; the underlying syslog-ng http destination does not support this
-feature.  Moreover, HEC Ack would significantly degrade performance for streaming data such as syslog.
+6. Add the following environment variables and values to `/opt/sc4s/env_file`:
+* Update `SC4S_DEST_SPLUNK_HEC_DEFAULT_URL` and `SC4S_DEST_SPLUNK_HEC_DEFAULT_TOKEN` to reflect the values for your environment. Do not configure HEC
+Acknowledgement when you deploy the HEC token on the Splunk side; syslog-ng http destination does not support this
+feature. 
 
 * The default number of `SC4S_DEST_SPLUNK_HEC_WORKERS` is 10. Consult the community if you feel the number of workers (threads) should
 deviate from this.
 
-* NOTE:  Splunk Connect for Syslog defaults to secure configurations.  If you are not using trusted SSL certificates, be sure to
-uncomment the last line in the example above.
-
+* Splunk Connect for Syslog defaults to secure configurations.  If you are not using trusted SSL certificates, be sure to
+uncomment the last line.
 
-## Dedicated (Unique) Listening Ports
-* NOTE:  Container networking differs on MacOS compared to that for linux.  On Docker Desktop, there is no "host" networking driver,
-so NAT networking must be used.  For this reason, each listening port on the container must be mapped to a listening port on the host.
-These port mappings are configured in the `docker-compose.yml` file or directly as a runtime option when run out of the CLI.
-Be sure to update the `docker-compose.yml` file or CLI arguments when adding listening ports for new data sources.
+## Create unique dedicated listening ports
+Docker Desktop does not support a "host" networking driver
```

Comment:
Docker Desktop on Linux supports it, but container networking differs on MacOS compared to that for linux